__tests__/app.test.js

```diff
@@ -192,11 +192,59 @@ describe("GET /api/articles", () => {
       .get("/api/articles")
       .expect(200)
       .then(({ body }) => {
-        console.log(body.articles);
         expect(body.articles).toBeSortedBy("created_at", {
           decending: true,
           coerce: true,
         });
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("responds with a array of comment objects of given id", () => {
+    const articleID = 5;
+    return request(app)
+      .get(`/api/articles/${articleID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        body.comments.forEach((comment) => {
```

Comment:
if we are using forEach - you should be testing the length of the array of comments beforehand.